### ISSUE TITLE
Notification created after clearAll() gets removed

### ISSUE THREAD

**Issue by tompau** [oblique-bit/oblique#17]
Hello, 

there seems to be issue with notification service/component when the notification created after notificationService.clearAll() get's removed: 

```
this.notificationService.success({ message: 'first message'});
this.notificationService.clearAll();
this.notificationService.success({ message: 'second message, shouldn\'t be closed, but it is ...', sticky: true});
this.notificationService.success({ message: 'third message, shouldn\'t be closed, working ...', sticky: true});
```
the second notification disappears after some delay, but the third one stays visible ...

**Comment by gillerr** [comment]
Hello

Thanks for the ticket, I have been able to reproduce the problem, It will be fixed the the next version

**Comment by gillerr** [comment]
Fixed with Oblique 6.1.0
